public/js/components/AtomList/AtomList.js

```diff
@@ -0,0 +1,173 @@
+import React, { PropTypes } from 'react';
+
+import {allAtomTypes} from '../../constants/atomData';
+import {atomTitleExtractor} from '../../util/atomTitleExtractor';
+import publishState from '../../util/publishState';
+import {supportedAtomTypes} from '../../constants/atomData';
+import {ManagedField} from '../ManagedEditor';
+import FormFieldTextInput from '../FormFields/FormFieldTextInput';
+import FormFieldCheckboxGroup from '../FormFields/FormFieldCheckboxGroup';
+import FormFieldSelectBox from '../FormFields/FormFieldSelectBox';
+import {Link} from 'react-router';
+
+class AtomList extends React.Component {
+
+    static propTypes = {
+        config: PropTypes.shape({
+            atomEditorUrls: PropTypes.shape({
+                explainer: PropTypes.string,
+                media: PropTypes.string
+            })
+        }),
+        atomListActions: PropTypes.shape({
+            getAtomList: PropTypes.func.isRequired
+        }).isRequired,
+        atomList: PropTypes.array
+    };
+    
+    state = {
+        params: {
+            types:[],
+            "page-size": "20",
+            q: "",
+            searchFields: "data.title,data.label,title,labels,data.body"
+        }
+    };
+    
+    componentWillMount() {
+        this.props.atomListActions.getAtomList(this.state.params);
+    }
+
+    updateAtomList = (newParams) => {
+        this.setState({
+            params: newParams
+        }, () => this.props.atomListActions.getAtomList(this.state.params));
+    };
+
+    getUserString = (changeRecord) => {
+        if (!changeRecord || !changeRecord.user) return 'unknown';
+        if (!changeRecord.user.firstName || !changeRecord.user.lastName) return changeRecord.user.email;
+        return `${changeRecord.user.firstName} ${changeRecord.user.lastName}`;
+    };
+
+    getDateString = (changeRecord) => {
+        if (!changeRecord) return 'unknown';
+        const date = new Date(changeRecord.date);
+        return `${date.toLocaleDateString()} ${date.toLocaleTimeString('en-GB', {formatMatcher: "basic"})}`;
```

Comment:
We have date-fns pulled into this project now, so that could be an option if you want to make this code a bit nicer?